src/hscore/hscore_rewards.c

```diff
@@ -471,21 +471,21 @@ local void flagWinCallback(Arena *arena, int freq, int *pts)
 			if (flagging_freq(arena, i->p_freq))
 				end_playtime(i, i->p_freq);
 
-		ticks_t q60_playtime = 1; // Mathematically 60th quartile, realistically 80th for small games (< 5)
+		ticks_t q60_playtime = 1; // Mathematically 60th percentile, realistically 80th for small games (< 5)
 		size_t n_flagging = 0;
 		FOR_EACH_PLAYER_IN_ARENA(i, arena)
-			if (flagging_freq(arena, i->p_freq) && i->p_freq == freq)
-				++n_flagging;	
+			if (flagging_freq(arena, i->p_freq) && i->p_freq == freq && IS_HUMAN(i))

```

Comment:
Should we be checking the ship here as well so we don't pick up spec/chatnet clients?